src/layman/rest_responses_test.py

```diff
@@ -58,26 +58,27 @@ def test_updated_at(publication_type):
 
 class TestResponsesClass:
     workspace = 'test_responses_workspace'
-    publication = 'test_responses_publication'
+    publication_layer = 'test_responses_publication_layer'
```

Comment:
Try to add publication type to ordering in prime db schema. Then this whole commit should not be needed,